### PR TITLE
apt: silence warnings when fetching apt-news

### PR DESCRIPTION
**Pull request by lucasmoura**
## Why is this needed?
When fetching the apt-news data, APT will try using the _apt user to store that information on the system. We are now creating a separate folder that is owned by _apt, which silences an APT warning when storing the apt-news data

Fixes: #3209
LP: #2070095

## Test Steps
Check that the APT news related tests are still working and that the error message mentioned on #3209 no longer appears on Noble

---

- [x] *(un)check this to re-run the checklist action*

### CODE REVIEW

**Review by orndorffgrant — CHANGES_REQUESTED** [review]
As part of our process to get ready for v35, I have rebased `next-v35` onto `main`. As a result, `next-v35` is now obsolete. Please rebase this on `main` and target the PR to `main`.

File: uaclient/defaults.py [hunk 1]
```diff
@@ -43,6 +43,10 @@
 ESM_APT_ROOTDIR = os.path.join(DEFAULT_DATA_DIR, PRIVATE_ESM_CACHE_SUBDIR)
 NOTICES_PERMANENT_DIRECTORY = os.path.join(DEFAULT_DATA_DIR, NOTICES_SUBDIR)
 NOTICES_TEMPORARY_DIRECTORY = os.path.join(UAC_RUN_PATH, NOTICES_SUBDIR)
+APT_UPGRADE_HOOK_ERR = os.path.join(
+    DEFAULT_DATA_DIR, "apt-upgrade-hook-err.txt"
+)
```
Comment (orndorffgrant) [inline uaclient/defaults.py:48]: I think this was brought in by mistake during the rebase and can be removed.
```suggestion
```

**Review by orndorffgrant — APPROVED** [review]
This worked in my test - thanks!

**Comment by lucasmoura** [comment]
@orndorffgrant I have fixed the rebase issue

**Comment by lucasmoura** [comment]
@julian-klode @panlinux If you can also please take a look at this change

**Review by julian-klode — APPROVED** [review]
lgtm

**Review by panlinux — APPROVED** [review]
+1

**Pull request merged**